paasta_tools/long_running_service_tools.py

```diff
@@ -107,6 +107,11 @@ def get_healthcheck_mode(self, service_namespace_config):
             raise InvalidHealthcheckMode("Unknown mode: %s" % mode)
         return mode
 
+    def get_bounce_priority(self):
+        """Gives a priority to each service instance which deployd will use to prioritise services
+        higher numbers are higher priority"""
+        return self.config_dict.get('bounce_priority', 0) * -1
```

Comment:
This -1 is going to cause confusion, but *not* giving -1 is going to cause even different confusion! :(

Add some docs in the RST file with examples to make this really clear